README.md

```diff
@@ -1,54 +1,54 @@
-# USP+ A universidade fora da sala de aula
+# USP+ The university outside the classroom
 
 [![Build Status](https://travis-ci.org/mrfelipenoronha/uspmais.svg?branch=master)](https://travis-ci.org/mrfelipenoronha/uspmais)
 [![Coverage Status](https://coveralls.io/repos/github/mrfelipenoronha/uspmais/badge.svg?branch=master)](https://coveralls.io/github/mrfelipenoronha/uspmais?branch=master)
 [![Heroku](https://heroku-badge.herokuapp.com/?app=uspmais&style=flat&svg=1)](https://uspmais.herokuapp.com)
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 
 
-![logo](logo.png "USP+  A universidade fora da sala de aula")
+![logo](logo.png "USP+  The university outside the classroom")
```

Comment:
Could you please add the link to the platform in the logo? I think it is just putting this whole line inside the description of a link tag